app/models/gram.rb

```diff
@@ -0,0 +1,53 @@
+class Gram < ActiveRecord::Base
+  # Associations
+  belongs_to :instagram_user
+
+  # Validations
+  validates :ig_id, uniqueness: true
+
+  # Methods
+  def self.collect_latest_posts(user)
+    followees = user.instagram_users
+
+    followees.each do |account|
+      last_post_id = account.grams.last.try :ig_id
+      unless last_post_id == nil
+        response = HTTParty.get("https://api.instagram.com/v1/users/#{account.ig_user_id}/media/recent/?min_id=#{last_post_id}&access_token=#{ENV['INSTAGRAM_ACCESS_TOKEN']}")
+      else # We have no posts of theirs on record
+        response = HTTParty.get("https://api.instagram.com/v1/users/#{account.ig_user_id}/media/recent/?count=5&access_token=#{ENV['INSTAGRAM_ACCESS_TOKEN']}")
+      end
+      posts = response["data"] # This is an array of hashes
+      posts.reverse # Changes it to ascending chronological order
```

Comment:
Using `.reverse` can lead to odd results unless you're running them against collections of primitives (numbers, strings). For hashes, it'd probably be safer to do something like this:
```
   posts.sort { |a, b| b[:timestamp] <=> a[:timestamp] }
```